src/components/Join.js

```diff
@@ -118,6 +119,10 @@ const Consent = compose(
               const { email, percentageFilled, ...rest } = loggedInUser;
               if (event.target.checked) {
                 setCustomStepMessage(null);
+                trackUserInteraction({
+                  category: 'Join',
```

Comment:
I wonder if we should export an object `constants` from `services/analyticsTracking`, that has categories / actions so we're not using magic strings when we track interactions.
```
import { constants } from 'services/analyticsTracking'
...
trackUserInteraction({
  category: constants.categories.join,
  action: constants.actions.acceptedTerms
});
```
?

Reply:
This is definitely on my todo list. Was just trying to squeeze some of the out before internal review so we can iterate on it once we go into ALPHA/BETA.